package.json

```diff
@@ -11,8 +11,8 @@
   "scripts": {
     "test": "jest app/javascript/ --coverage",
     "test:watch": "jest app/javascript/ --watch",
-    "storybook": "start-storybook -p 6006 -c app/javascript/.storybook -s app/javascript/.storybook/assets",
```

Comment:
We need all the assets that we have in the application, so using this as the static assets folder for Storybook instead of copying assets over.